interfaces/payment-request.idl

```diff
@@ -27,6 +27,8 @@ dictionary PaymentMethodData {
 dictionary PaymentCurrencyAmount {
   required DOMString currency;
   required DOMString value;
+  // Note: currencySystem is "at risk" of being removed!
```

Comment:
Looks like data source isn't fresh. This is gone from the spec and I removed it manually in https://github.com/web-platform-tests/wpt/pull/11099.